matrix/lambdas/filter_merge/driver.py

```diff
@@ -1,2 +1,34 @@
-def driver():
-    pass
+import typing
+
+from ...common.dynamo_handler import DynamoHandler
+from ...common.lambda_handler import LambdaHandler
+from ...common.lambda_handler import LambdaName
+
+
+class Driver:
+    """
+    The first task in a distributed filter merge job.
+    """
+    def __init__(self):
+        self.lambda_handler = LambdaHandler()
+        self.dynamo_handler = DynamoHandler()
+
+    def run(self, request_id: str, bundle_fqids: typing.List[str], format: str):
+        """
+        Initialize a filter merge job and spawn a mapper task for each bundle_fqid.
+
+        :param request_id: Filter merge job request ID
+        :param bundle_fqids: List of bundle fqids to be queried on
+        :param format: MatrixFormat file format of expression matrices
+        """
+        self.dynamo_handler.put_state_item(request_id, len(bundle_fqids))
+        self.dynamo_handler.put_output_item(request_id)
```

Comment:
(nit) any better names here for put_state_item and put_output_item?

Reply:
How's `create_state_table_entry` and `create_output_table_entry`?